sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/IssueLocationCollector.cs

```diff
@@ -116,7 +116,7 @@ public static bool IsNotIssueLocationLine(string line)
             var patternWithoutLineStart = PRECISE_ISSUE_LOCATION_PATTERN.Substring(1);
             if (Regex.IsMatch(lineText, patternWithoutLineStart))
             {
-                Execute.Assertion.FailWith("Line matches expected location pattern, but doesn't start at the beginning of the line.");
+                Execute.Assertion.FailWith($"Line matches expected location pattern, but doesn't start at the beginning of the line. Line: {line.LineNumber}");
```

Comment:
Miscellaneous: having the line number speeds up fixing formatting issues in the tests.